tests/unit/executors/indexers/test_numpyindexer.py

```diff
@@ -160,3 +160,25 @@ def test__get_sorted_top_k(batch_size, num_docs, top_k):
 
         np.testing.assert_equal(idx, expected_idx)
         np.testing.assert_equal(dist, expected_dist)
+
+
+@pytest.mark.parametrize('batch_size, compress_level', [(None, 0), (None, 1), (2, 0), (2, 1)])
+def test_numpy_indexer_empty_data(batch_size, compress_level, test_metas):
+    np.random.seed(500)
+    num_dim = 64
+    num_query = 10
+    query = np.array(np.random.random([num_query, num_dim]), dtype=np.float32)
+
+    idx_file_path = os.path.join(test_metas['workspace'], 'np.test.gz')
+    with NumpyIndexer(index_filename=idx_file_path, compress_level=compress_level, metas=test_metas) as indexer:
+        indexer.batch_size = batch_size
+        indexer.touch()
+        indexer.save()
+        assert os.path.exists(indexer.index_abspath)
+        save_abspath = indexer.save_abspath
+
+    with BaseIndexer.load(save_abspath) as indexer:
+        assert isinstance(indexer, NumpyIndexer)
+        idx, dist = indexer.query(query, top_k=4)
+        assert idx == None
+        assert idx == None
```

Comment:
Fix the duplicates.
```
assert dist == None
```

Reply:
https://github.com/jina-ai/jina/pull/1396